### DISCUSSION TITLE
Chip (badge) select

### DISCUSSION THREAD

**Discussion by dsm-72** [saadeghi/daisyui#2002] [Ideas / feature requests]
Please :)

**Comment by saadeghi** [comment]
what do you mean `select`?  
There is [`badge`](https://daisyui.com/components/badge/) if that's what you need.

**Comment by dsm-72** [comment]
@saadeghi see:
- https://vuetifyjs.com/en/components/chip-groups/#mandatory
- https://vuetifyjs.com/en/components/chips/#in-selects
- https://vuetifyjs.com/en/components/selects/#chips

**Comment by saadeghi** [comment]
To make them selectable, you can use checkboxes or radio buttons with `btn` class like this:  
https://play.tailwindcss.com/tWyuMCyf8E?layout=horizontal

But something like this https://vuetifyjs.com/en/components/chips/#in-selects or https://vuetifyjs.com/en/components/selects/#chips is not possible with CSS. You can use dropdown, checkbox, and badge to make something like this but you also need JS to add/remove items. CSS can't do this for you.

**Comment by thiagoprazeres** [comment]
The new version 5 has the filter component which is to simulate the chips:
https://daisyui.com/components/filter/
